.travis.yml

```diff
@@ -5,10 +5,10 @@ cache: yarn
 
 # https://github.com/nodejs/Release
 node_js:
-  - '14'
-  - '13'
-  - '12'
-  - '10'
+  - 10
+  - 12
+  - 14
+  - node
```

Comment:
AFAIK Travis will run duplicate versions so we will have duplicating runs of v14.

Reply:
At this moment it's indeed duplicate, but whenever v15 comes out, it's automatically tested too

Reply:
Make sense however I do all commits only through separate PRs so CI time is critical for this workflow.